rootfs-builder/clearlinux/config.sh

```diff
@@ -26,6 +26,6 @@ PACKAGES="util-linux-bin iptables-bin libudev0-shim chrony"
 INIT_PROCESS=systemd
 # List of zero or more architectures to exclude from build,
 # as reported by  `uname -m`
-ARCH_EXCLUDE_LIST=(ppc64le)
+ARCH_EXCLUDE_LIST=( aarch64 ppc64le )
```

Comment:
For completeness, I wonder if it's also worth adding `s390x`?

Reply:
yep, you are right. ;)
and also for completeness, the latest euleros v2.3 only supports amd64 and arm64. you could check it from their official [docker hub repo](https://hub.docker.com/_/euleros)